src/components/common/search/search.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import { Button, Radio, Input } from 'antd';
+import React from "react";
```

Comment:
Also the appropriate name of the file is SearchSection

Reply:
Fixed